### PR TITLE
adding Riigikogu members

### PR DESCRIPTION
**Pull request by bgmello**
Solving the issue https://github.com/opensanctions/crawler-planning/issues/12

### CODE REVIEW

File: datasets/ee/riigikogu/ee_riigikogu.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Members of the Riigikogu
```
Comment (jbothma) [inline datasets/ee/riigikogu/ee_riigikogu.yaml:1]:
```suggestion
title: Estonia Members of the Riigikogu
```

File: datasets/ee/riigikogu/crawler.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    if occupancy is None:
+        return
+
+    context.emit(entity, target=True)
+    context.emit(position)
+    context.emit(occupancy)
+    
+
+def crawl(context: Context):
+    """
```
Comment (jbothma) [inline datasets/ee/riigikogu/crawler.py:87]: I think this function is clear enough without a comment. No need to document the params and design of every function, especially `crawl`, every time. Especially with nice clear variable names like members_url and member_url

Now that I read the code, I see members_url is a list - so perhaps make that `member_urls`, then it's nice and clear what's going on even without comments

File: datasets/ee/riigikogu/crawler.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    :return: The URLs for the pages.
+    """
+
+    try:
+        main_website = context.fetch_html(context.data_url)
+
+    except:
+        context.log.error("Couldn't fetch main website")
+        return None
+
```
Comment (jbothma) [inline datasets/ee/riigikogu/crawler.py:19]: no need for you to handle exceptions from http requests generally - an error in fetch_html will raise an exception handled by the crawler runner, resulting in an error log and alert on the website at https://www.opensanctions.org/issues/

File: datasets/ee/riigikogu/crawler.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    entity = context.make("Person")
+    entity.id = context.make_id(name)
+
+    try:
+        # the phone number apparently is the first contact in the list
+        # but to make sure we correctly get it, we based our xpath on the icon
+        phone_number = member_page_html.xpath('//*[@class="icon icon-tel"]/../text()')[0].replace(" ", "")
+        entity.add("phone", phone_number)
+    except:
+        context.log.warning("Couldn't find phone number")
```
Comment (jbothma) [inline datasets/ee/riigikogu/crawler.py:58]: are we currently getting these warnings or is this just in case they change things in future?

**Review by jbothma — CHANGES_REQUESTED** [review]
Nice! Just a few small things

File: datasets/ee/riigikogu/crawler.py (hunk 4 above)
Comment (bgmello) [reply datasets/ee/riigikogu/crawler.py:58]: No, currently everyone has a e-mail and phone number. It's just in case it changes in the future

Comment (jbothma) [reply datasets/ee/riigikogu/crawler.py:32]: Lovely, thanks!

**Pull request merged**